portfolio_ahardman_stylesheet.css

```diff
@@ -1,261 +1,339 @@
-}
-/* footer */
+	display: flex;
+	align-items: center;
+	font-size: 3rem;
+	padding: 0px;
+	margin: 0px;
+}
+.about_me .p_container{
+	display: flex;
+	flex-wrap: wrap;
+	margin-top: 20px;
+	width: 100%;
+}
+.p_container p{
+	width: 100%;
+	margin-top: 40px;
+	padding: 0px;
+	font-size: 1.5rem;
+}
+/* ----------------------footer---------------------- */
 footer{
-	width:100%;
-	height:150px;
-	display:flex;
-	justify-content:center;
-	align-items:center;
-	border:1px solid blue;
+	width: 100%;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	background-image: linear-gradient(to right,rgb(150,150,150),rgb(180,180,180));
 }
 footer .center.container{
-	display:flex;
-	justify-content:center;
-	width:70%;
-	height:100%;
-	background-color:green;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	padding: 1vw;
 }
 footer h3{
-	display:flex;
-	align-items:center;
-	font-size:2rem;
-	margin:0px;
-	padding:0px;
-	background-color:gray;
+	display: flex;
+	align-items: center;
+	text-align: center;
+	font-size: 1.5rem;
+	margin: 0px;
+	padding: 0px;
 }
 footer .center.container .right.container{
-	display:flex;
-	justify-content:flex-start;
-	align-items:space-around;
-	flex-wrap:wrap;
-	margin-left:40px;
-	width:50%;
+	display: flex;
+	justify-content: flex-start;
+	align-items: center;
+	margin-left: 2vw;
 }
 form{
-	display:flex;
-	align-items:center;
-	width:100%;
-	height:50%;
-	background-color:yellow;
+	display: flex;
+	align-items: center;
+	width: 100%;
 }
 .email_input{
-	height:60%;
-	width:80%;
-	margin-right:10px;
-	font-size:1rem;
-	border-radius:5px;	
-	border-color:black;
+	margin-right: 1vw;
+	font-size: 1rem;
+	padding: 0.5vw 0.5vw 0.5vw 1vw;
+	background-color: rgb(230,230,230);
+	border-radius: 5px;	
+	border: 2px solid rgb(0, 164, 153);
+}
+.email_input:focus{
+	font-size: 1rem;
+	padding-left: 1vw;
+	border: 2px solid rgb(0, 164, 153);
+	background-color: rgb(230,230,230);
 }
 form button{
-	height:60%;
-	width:20%;
-	border-radius:5px;
-	font-size:0.8rem;	
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	padding: 0.5vw;
+	border-radius: 5px;
+	border: 2px solid rgb(0, 164, 153);
+	background-color: rgb(230,230,230);
+	font-size: 1rem;
+	margin-right: 1vw;	
 }
 form .email_input::placeholder{
-	font-size:1rem;
-	padding:10px;
-	font-style:italic;
+	font-size: 1rem;
+	background-color: rgb(230,230,230);
+	font-style: italic;
+}
+.email_input:focus::placeholder{
+	font-size: 0px;
 }
 footer a{
-	height:45px;
-	width:45px;
-	display:flex;
-	justify-content:center;
-	align-items:center;
-	background-color:red;
-	margin:0px 20px 0px 0px;
-	padding:0px;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	margin: 0vw 0vw 0vw 2vw;
+	padding: 0px;
+}
+footer a img{
+	height: 40px;
+	width: 40px;
+	filter: grayscale(100%);
+}
+.footer_links{
+	display: flex;
+	flex-direction: row;
+}
+/* mobile responsivity */
+@media only screen and (max-width: 600px) {
```

Comment:
nav bar links when below like 400 pixels width starts to cover eachother. tbh not sure this really matters.